lib/MatterbridgeManager.php

```diff
@@ -342,7 +342,7 @@ private function generateConfig(array $bridge): string {
 					$serverUrl = $part['server'];
 				} else {
 					$serverUrl = preg_replace('/\/+$/', '', $this->urlGenerator->getAbsoluteURL(''));
-					$content .= "	SeparateDisplayName = true" ."\n";
+					$content .= '	SeparateDisplayName = true' ."\n";
```

Comment:
`"\n"` is fine for standard, or also should be changed?

Reply:
It needs to be, otherwise it’s not a new line anymore